rental.rb

```diff
@@ -17,8 +17,6 @@ def initialize(date, book, person)
     person.rental << self
   end
 
```

Comment:
- [ ] I am not able to create a `rental` number 5
![rental](https://user-images.githubusercontent.com/44745855/180620266-aa2d6cda-f6f3-4dba-b0d4-1dbfca42eeac.png)